lib/SetAPI/assembly/AssemblySetInterfaceV1.py

```diff
@@ -1,82 +1,138 @@
+from SetAPI.error_messages import (
+    data_required,
+    include_params_valid,
+    items_list_required,
+    ref_must_be_valid,
+    ref_path_must_be_valid,
+    ref_required,
+)
```

Comment:
Put all the standard error messages into their own module so it's easy to check that everything is reporting errors in the same way